src/utils/platform.js

```diff
@@ -0,0 +1,11 @@
+const platform = (() => {
+  if ('userAgentData' in navigator && 'platform' in navigator.userAgentData && navigator.userAgentData.platform) {
+    return navigator.userAgentData.platform
+  }
+  const userAgent = navigator.userAgent.toLowerCase()
+  if (userAgent.includes('linux')) return 'Linux' // includes Android
+  if (userAgent.includes('win32') || userAgent.includes('windows')) return 'Windows'
+  return null
+})()
+
+export const isWindows = !platform || platform === 'Windows'
```

Comment:
if we don't know which platform, we should probably still show the windows instructions